### PR TITLE
Translate ServiceList to yaml

### PR DESCRIPTION
**Pull request by openBackhaul**

### CODE REVIEW

File: OamLog+forwardings.yaml [hunk 1]
```diff
@@ -0,0 +1,518 @@
...
+    management-requests:
+      operation-client-update:
+      fc-port-update:
+      fc-port-deletion:
+      operation-client-deletion:
+    initiating-requests:
+      - server-name: /v1/bequeath-your-data-and-die
+        uuid: ol-1-0-0-op-s-im-000
+    consequent-requests:
+      - client-name: NewRelease:://v1/record-service-request
```
Comment (IswaryaaS) [inline OamLog+forwardings.yaml:161]: the consequent-request to be updated to point to NewRelease://v1/record-oam-request, ol-1-0-0-op-c-im-nr-1-0-0-001

**Review by IswaryaaS — CHANGES_REQUESTED** [review]

File: OamLog+forwardings.yaml [hunk 2]
```diff
@@ -0,0 +1,518 @@
...
+        - server-name: /v1/disregard-application
+          uuid: ol-1-0-0-op-s-is-002
+      operation-client-deletion:
+        - server-name: /v1/disregard-application
+          uuid: ol-1-0-0-op-s-is-002
+    initiating-requests:
+        - server-name: /v1/regard-application
+          uuid: ol-1-0-0-op-s-is-001
+    consequent-requests:
+      - client-name: RegistryOffice:://v1/redirect-oam-request-information
```
Comment (IswaryaaS) [inline OamLog+forwardings.yaml:503]: As per my understanding, /v1/redirect-oam-request-information (operation-client under each application) is applicable only for OL application, changing the api-segment of all occurences of /v1/redirect-oam-request-information to "individual management (im)" would be more precise.

File: OamLog+services.yaml [hunk 3]
```diff
@@ -30,6 +30,8 @@ servers:
           uuid: ol-1-0-0-op-s-1000
     service:
       basic:
+        - service-name: /v1/start-application-in-generic-representation
+          uuid: ol-1-0-0-op-s-2000
         - service-name: /v1/inform-about-application
           uuid: ol-1-0-0-op-s-2001
```
Comment (IswaryaaS) [inline OamLog+services.yaml:36]: As per my understanding, /v1/redirect-oam-request-information (operation-client under each application) is applicable only for OL application, moving this service to "individual management (im)" would be more precise.

**Review by IswaryaaS — CHANGES_REQUESTED** [review]

File: OamLog+services.yaml (hunk 3 above)
Comment (openBackhaul) [reply OamLog+services.yaml:36]: Changed as proposed.

File: OamLog+forwardings.yaml [hunk 4]
```diff
@@ -0,0 +1,518 @@
...
+        uuid: ol-1-0-0-op-c-bs-ro-1-0-0-000
+      - client-name: TypeApprovalRegister:://v1/redirect-oam-request-information
+        uuid: ol-1-0-0-op-c-bs-tar-1-0-0-000
```
[the remark this answers is not in the document]
Comment (openBackhaul) [reply OamLog+forwardings.yaml:506]: Changed as proposed.

File: OamLog+forwardings.yaml [hunk 5]
```diff
@@ -0,0 +1,518 @@
...
+    management-requests:
+      operation-client-update:
+        - server-name: /v1/redirect-oam-request-information
+          uuid: ol-1-0-0-op-s-bm-003
+      fc-port-update:
+        - server-name: /v1/redirect-oam-request-information
+          uuid: ol-1-0-0-op-s-bm-003
+      fc-port-deletion:
+      operation-client-deletion:
+    initiating-requests:
```
[the remark this answers is not in the document]
Comment (openBackhaul) [reply OamLog+forwardings.yaml:112]: Changed as proposed.

File: OamLog+services.yaml [hunk 6]
```diff
@@ -30,6 +30,8 @@ servers:
           uuid: ol-1-0-0-op-s-1000
     service:
       basic:
+        - service-name: /v1/start-application-in-generic-representation
+          uuid: ol-1-0-0-op-s-2000
         - service-name: /v1/inform-about-application
```
[the remark this answers is not in the document]
Comment (openBackhaul) [reply OamLog+services.yaml:35]: Changed as proposed.

File: OamLog+forwardings.yaml (hunk 1 above)
Comment (openBackhaul) [reply OamLog+forwardings.yaml:161]: Changed as proposed.

File: OamLog+forwardings.yaml [hunk 7]
```diff
@@ -0,0 +1,520 @@
...
+      # [all oam requests]
```
Comment (IswaryaaS) [inline OamLog+forwardings.yaml:113]: Along with the new add-ons, can we add a comment like [all oam put requests] to "initiating-requests" of OamRequestCausesLtpUpdateRequest forwarding

File: OamLog+forwardings.yaml [hunk 8]
```diff
@@ -0,0 +1,520 @@
...
+        - server-name: /v1/disregard-application
+          uuid: ol-1-0-0-op-s-is-002
+      operation-client-deletion:
+        - server-name: /v1/disregard-application
+          uuid: ol-1-0-0-op-s-is-002
+    initiating-requests:
+        - server-name: /v1/regard-application
+          uuid: ol-1-0-0-op-s-is-001
+    consequent-requests:
+      - client-name: RegistryOffice:://v1/redirect-oam-request-information
```
Comment (IswaryaaS) [inline OamLog+forwardings.yaml:505]: uuid of RegistryOffice:://v1/redirect-oam-request-information to be updated to ol-1-0-0-op-c-im-ro-1-0-0-004

**Review by IswaryaaS — CHANGES_REQUESTED** [review]

File: OamLog+forwardings.yaml (hunk 8 above)
Comment (openBackhaul) [reply OamLog+forwardings.yaml:505]: Changed as proposed. Many thanks for indicating the mistake.

Comment (IswaryaaS) [reply OamLog+forwardings.yaml:505]: Somehow the update is not reflecting for RegistryOffice:://v1/redirect-oam-request-information

**Review by IswaryaaS — CHANGES_REQUESTED** [review]
recent commit is missing for update requested in https://github.com/openBackhaul/OamLog/pull/66#discussion_r1021353518

**Pull request merged**